benchmarks/benchmarks/benchmarks.py

```diff
@@ -173,11 +174,11 @@ class NUTSInitSuite:
     def time_glm_hierarchical_init(self, init):
         """How long does it take to run the initialization."""
         with glm_hierarchical_model():
-            pm.init_nuts(init=init, chains=self.chains, progressbar=False)
+            pymc3.step_methods.hmc.nuts.init_nuts(init=init, chains=self.chains, progressbar=False)
 
     def track_glm_hierarchical_ess(self, init):
         with glm_hierarchical_model():
-            start, step = pm.init_nuts(
+            start, step = pymc3.step_methods.hmc.nuts.init_nuts(
```

Comment:
init nuts seemed weird in nthe top level namespace so I opted for this. Im not sure if 
1. Want it in the top level namespace?
2. If not what namespace is most appropriate

Defaulting to most verbose until someone more opinionated tells me their opinion

Reply:
Sounds fine, but you can import at the top as `from pymc3.step_methods.hmc.nuts import init_nuts`.

Also we could provide access to the more user-facing methods directly under `step_methods`, so that they can do `pym3.step_methdos.nuts`/ `pymc3.step_methods.hmc` / `pymc3.step_methods.nuts.init_nuts`